onnxruntime/core/providers/iree/compiler/torch-mlir-import-onnx/OnnxImporter.cpp

```diff
@@ -387,6 +387,14 @@ ContextCache::ConvertTensorProtoToAttr(const onnx::TensorProto &tp) {
     case onnx::TensorProto::DataType::TensorProto_DataType_FLOAT:
       return mlirDenseElementsAttrFloatGet(tensor_type, tp.float_data_size(),
                                            tp.float_data().data());
+    case onnx::TensorProto::DataType::TensorProto_DataType_INT8: {
+      std::vector<int8_t> int8_conversion;
+      int8_conversion.reserve(tp.int32_data_size());
+      for (uint64_t v : tp.int32_data())
```

Comment:
Why is it `uint64_t`'s in the `tp.int32_data()`?

Should this be 
```suggestion
      for (int32_t v : tp.int32_data())
```
?

Reply:
@zjgarvey Yep, that is a correct observation. It should not be `uint64_t`. If the value is negative on the RHS, it could cause an explosion when it takes the complement.

Reply:
Thanks for noticing this @zjgarvey @vinayakdsci, fixed this.